VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/hearing-layout.component.html

```diff
@@ -1,12 +1,26 @@
+<div class="govuk-radios__item layoutradio1" *ngIf="showInput">
+  <input
+    [checked]="isSelected"
+    [value]="layout"
+    (change)="emitSelected()"
+    class="govuk-radios__input"
+    id="layout-radio-button-{{layout}}"
+    name="layout"
+    type="radio"
+    attr.aria-labelledby="{{headerLayoutTitlePrefix}}{{layout}}"
+    attr.aria-describedby="{{headerLayoutDescriptionPrefix}}{{layout}}"
+  />
+  <label class="govuk-label govuk-radios__label"></label>
```

Comment:
We should remove this label. I know it makes them display wrong but we should handle that with CSS on the input rather that relying on an empty label.